examples/chart/teleport-cluster/values.yaml

```diff
@@ -15,6 +15,9 @@ kubeClusterName: ""
 # Values that you may need to change.
 ##################################################
 
+# Additional labels to add into metadata of all resources
+additionalLabels: {}
```

Comment:
Please move this down to the `Extra Kubernetes Configuration` section, around line 166.

Reply:
@nlamirault If you could handle this request, we'll get this approved and merged. Thanks!